nl_server/loader.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+GCS_BUCKET = 'datcom-nl-models'
```

Comment:
Shouldn't this also be provided as a config (because what if we decide to change this folder or rename it some day)? Could this be set in the model.yaml?

Reply:
The config holds values that changes often. Since this would be fairly stable (if not changed). We can keep it as a constant here.

Reply:
ok, sgtm